downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -246,6 +246,59 @@ public function main() returns error? {
 
 - Added parameter passing support for XSLT transformations.
 
+### Connector updates
+
+New versions of the following connectors have been released with major updates, as part of the Ballerina connector revamp initiative.
+All listed connectors have been released under new major versions, featuring significant API and functionality changes, along with improved documentation and examples.
+
+#### `asb` package
+
+- Introduced support for listener-service-based async message consumption functionality.
+
+#### `dayforce` package
+
+- Introduced as a new connector with support for the Dayforce REST API v1.
+
+#### `discord` package
+
+- Introduced as a new connector with support for the Discord REST API v10.
+
+#### `salesforce` package
+
+- Added support for Salesforce listener with Streaming API for real-time data synchronization and event-driven workflows.
+- Added support for base types with `salesforce.types` submodule.
+- Reorganized salesforce clients for better usability.
+
+#### `sap` package
+
+- Added support for invoking any OData endpoints in S/4HANA.
+- Added support for automatic CSRF token authentication for enhanced security and usability.
+
+#### `sap.jco` package
+
+- Enabled invocation of RFC modules with improved type binding.
+- Added support for sending and receiving iDocs.
+
+#### `sap.s4hana.sales` packages
+
+- Provided a high-level interface for APIs related to Sales and Distribution (SD) in S/4HANA.
+- Provided operations for sales order management, delivery processing, and billing.
+
+#### `slack` package
+
+- Replaced existing remote function-based APIs with resource function-based APIs.
```

Comment:
We usually use code blocks for keywords, right?

```suggestion
- Replaced existing `remote` method-based APIs with `resource` method-based APIs.
```

Reply:
Here I want to highlight the concepts of `remote methods` and `resource methods` in Ballerina (and not the qualifiers), so I'm not sure if highlighting only the keywords makes much sense in this case. @MaryamZi thoughts?

Reply:
Yeah, in this scenario I think we can skip too, since they make sense without backticks too.